packages/paste-core/components/anchor/__tests__/anchor.test.tsx

```diff
@@ -9,6 +9,15 @@ describe('Anchor', () => {
     expect(asFragment()).toMatchSnapshot();
   });
 
+  it('it should render an inverse anchor', () => {
```

Comment:
Let's omit snapshotting component renders from now on as a rule. Since we have VRT (applitools and chromatic) this now becomes noise. As long as we add a story, that now becomes the test.